lib/Zonemaster/CLI.pm

```diff
@@ -389,6 +381,22 @@ sub run {
         Zonemaster::Engine::Profile->effective->merge( $profile );
     }
 
+    if ( defined $self->sourceaddr4 ) {
+        eval { Zonemaster::Engine::Profile->effective->set( q{resolver.source4}, $self->sourceaddr4 ); };
+        if ( $@ ) {
+            say STDERR __x( "Error: invalid value for --sourceaddr4: {reason}", reason => $@ );
```

Comment:
Is the `reason` argument also translated according to the user’s locale?

Reply:
It depends on the OS and the OS locale setting if `$@` is translated or not, doen't it?

Reply:
No, it's a plain English string. I created zonemaster/zonemaster-engine#1375 to internationalize it.

Reply:
@matsduf But I believe you're right that errors reported by the OS (and possibly other dependencies) may be internationalized.